.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,41 @@
+name: Generate MDX and Update Mintlify Documentation
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Let's change this to workflow_dispatch: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch